src/components/forms/formr-part-b/viewSections/ViewSectionTestHelper.tsx

```diff
@@ -0,0 +1,112 @@
+import { mount } from "@cypress/react";
```

Comment:
Ugly, since its not obviously used, but otherwise all the references to `cy` threw up errors.

Reply:
@ReubenRobertsHEE I think adding this to the top of the file instead should do the trick: 
`// load type definitions that come with Cypress module`
`/// <reference types="cypress" />`

Reply:
Yeah it doesn't look good. Learning from others code from the project, I find `/// <reference types="cypress" />` at the top of the file would help this.

Reply:
Nice - thanks @john12321 and @DorisWongNHS